test/unit/routes/apply/eligibility/new-claim/test-same-journey-as-last-claim.js

```diff
@@ -53,6 +53,16 @@ describe('routes/apply/eligibility/new-claim/same-journey-as-last-claim', functi
         })
     })
 
+    it('should respond with a 302', function () {
```

Comment:
expand description of test case so its clear to someone why it does this (documents the code)
`should respond with a 302 when no expenses and cannot duplicate claim`